### DISCUSSION TITLE
Make an option to place the New Tab "+" right under the opened tabs

### DISCUSSION THREAD

**Discussion by benstone326** [zen-browser/desktop#2220] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Please make an option to place the New tab "+" under the opened tabs, just like in Ede.
![image](https://github.com/user-attachments/assets/d8e92417-2d02-455f-bc1e-27d6935d1c0b)

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a9

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1999/make-an-option-to-place-the-new-tab-right-under-the-opened-tabs">ZEN-1999 Make an option to place the New Tab "+" right under the opened tabs</a></p>
